BRANDING.md

```diff
@@ -0,0 +1,55 @@
+# Istio Brand Guidelines
+ 
+The Istio trademark and logos (“Istio Trademarks”) are treated separately from the copyright or patent license grants contained in the Apache-licensed Istio repositories on GitHub. Any use of the Istio Trademarks other than those permitted in these guidelines must be approved in advance.
+
+## Purpose of the Brand Guidelines
+    
+These guidelines exist to ensure that the Istio project can share its platform under open source licenses while making sure that the “Istio” brand is protected as a meaningful source identifier in a way that is consistent with trademark law.
+ 
+## Acceptable Uses
+
+Given the open nature of Istio, you may use the Istio trademark to refer to the project without prior written permission. Examples of these acceptable references include the following:
+
+-   To refer to the Istio project itself;
+    
+-   To refer to unmodified source code or other files shared by the Istio repositories on GitHub;
+    
+-   To accurately identify that your design or implementation is based on or is for use with the Istio platform.
+    
+-   To discuss the Istio project for educational or informational purposes
+    
+*Examples:*
+
+-   “[Your Product/Offering] for Istio.”
+    
+-   “[XYZ] Conference for Istio Users.”
+    
+-   “Making [XYZ] better with Istio”
+    
+## Usage Guidelines
+
+-   The Istio name may not be used or registered in a manner that would cause confusion as to the Istio project’s affiliation or endorsement.
+    
+-   Don’t use the Istio name as part of your company name, product name, domain name, or social media profile.
+    
+-   Other than as permitted by these guidelines, the Istio name should not be combined with other trademarks, terms, or source identifiers.
+    
+-   Don't remove, distort or alter any element of the Istio Trademarks without permission:
+    
+-   Do not shorten, abbreviate, or change the spelling of the Istio name, for example in mass communication, product names, social media profiles, domain names or event names).
```

Comment:
The "for example" is awkward to me. Does the form in which this "change" appears matter? Can't we just leave it at "Do not shorten, abbreviate, or change the spelling of the Istio name" ?